src/main/resources/resources/logisim/strings/gui/gui_zh.properties

```diff
@@ -216,8 +216,8 @@ exportEmptyCircuitsMessage = 没有非空线路可供导出。
 exportEmptyCircuitsTitle = 无法导出
 exportGifFilter = GIF文件(*.gif)
 exportImageDirectorySelect = 选择导出目录
-exportImageProgress = 正在计算图像.
-exportImageSelect = 导出图像
+exportImageProgress = 正在计算图片...
```

Comment:
As we are using Unicode, please use consistently `…` (horizontal ellipsis) instead of `...` (three consecutive `.`). This may not make a visible difference for the font you are using, but a noticeable one for others. Also, it avoids efficiently avoids unintended line breaks in the middle of the ellipsis.

Reply:
Even though it will not be an issue for Windows users, I have found minor difference in other fonts. But I doubt this change, because:
- Strings of ALL other languages (including English) uses "..." (three consecutive `.`) instead of "..." (U+2026)
   - I searched the localization folder and found zero instances of U+2026
- In recent years most Chinese apps do not use ellipsis in menus anymore, so there are very few instances to be found. Even Microsoft dropped them in many cases. Many of the apps that still has one are currently using uses three consecutive `.`.
- No strings, as found, are long enough to be affected by line-break. All of them are in context menus, where whey are fully able to automatically adapt to the width of the rendered texts, and is not limited by the width of the window.

I have prepared those changes but I would request further reason before actually commiting.